include/event_dbus_monitor.hpp

```diff
@@ -243,6 +245,16 @@ void registerVMIIPChangeSignal()
         VMINwPropertyChange);
 }
 
+void registerVMIEthIntfPropChangeSignal()
```

Comment:
This D-bus signal hits on IP address, gateway or any other property change on this interface as well right?

so with this change, it sends redfish events on any property change